dbt_subprojects/dex/models/automated_trades/dex_automated_base_trades.sql

```diff
@@ -1,18 +1,17 @@
 {{ config(
     schema = 'dex'
     , alias = 'automated_base_trades'
-    , partition_by = ['block_month', 'blockchain', 'project']
+    , partition_by = ['block_month', 'blockchain']
     , materialized = 'incremental'
     , file_format = 'delta'
     , incremental_strategy = 'merge'
-    , unique_key = ['blockchain', 'project', 'version', 'tx_hash', 'evt_index']
+    , unique_key = ['blockchain', 'block_number', 'tx_index', 'evt_index']
     , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
     )
 }}
 
 {% set models = [
-    ref('dex_ethereum_automated_base_trades')
-    , ref('dex_arbitrum_automated_base_trades')
+    ref('dex_multichain_automated_base_trades')
```

Comment:
`dex_automated_base_trades` probably won't ever add models here, since you add to the `multichain` one instead? should we remove the for loop / union logic here then to simplify?
or can it be taken even a step further, where you only need one or the other between this model and `dex_multichain_automated_base_trades`? why store the data twice?